tools/migrator/src/main/java/org/wso2/carbon/ei/migration/MigrationClientException.java

```diff
@@ -0,0 +1,11 @@
+package org.wso2.carbon.ei.migration;
+
+public class MigrationClientException extends Exception {
```

Comment:
Add Licence headers and doc comments.

Reply:
I have added the licence headers and doc comments.